From ea25995d8611e2969cd97d5967918895793441b0 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Tue, 31 Aug 2021 07:08:21 +0530
Subject: [PATCH] Update security_patch_level string

Update security_patch_level as the oct ASB
pacthes are integrated

Signed-off-by: svenate <salini.venate@intel.com>
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 13b85c0663..084f400e67 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -240,7 +240,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2021-07-05
+      PLATFORM_SECURITY_PATCH := 2021-10-01
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.17.1

